lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -1521,17 +1521,20 @@ class LineChartPainter extends AxisChartPainter<LineChartData> {
       }
     }
 
+    touchedSpots.sort((a, b) => a.distance.compareTo(b.distance));
```

Comment:
I don't think changing the order of 'touchedSpots' is a problem here?

Reply:
I don't think too. It looks good to me.